storage/database/src/tests.rs

```diff
@@ -0,0 +1,35 @@
+//! Utilities for `cuprate_database` testing.
+//!
```

Comment:
None of the tests were changed, just kept database ones in `cuprate_database` and moved blockchain ones to `cuprate_blockchain`.